openshift37/assets/enable-lio.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
I understand why the script is here, but I would prefer not to be duplicating this script across every tutorial. It makes it more difficult to keep these tutorials updated in the future.

Reply:
ok